values.yaml

```diff
@@ -104,3 +104,52 @@ oda:
   use_hostpath: true
   ddaQueue: queue-osa11
   ddcache: "/net/cdcicn02/scratch/shared/savchenk/data/reduced/ddcache"
+
+plugins:
+  osa:
+    ebabled: true
+    instruments:
+      isgri:
+        dispatcher_mnt_point: /data
+        data_server_cache: reduced/ddcache
+        dummy_cache: /data/dummy_prods
+        data_server_url: http://oda-dda-interface:8000
+
+      jemx:
+        dispatcher_mnt_point: /data
+        data_server_cache: reduced/ddcache
+        dummy_cache: /data/dummy_prods
+        data_server_url: http://oda-dda-interface:8000
+
+  magic:
+    enabled: true
+    instruments:
+      magic:
+        data_server_url: http://oda-magic:8000
+        dummy_cache: /data/dummy_prods
```

Comment:
what is point of making dummy_products configurable if they have to necessarily agree with what is built in the container? Generally in the deployment one does not provide the dummy products, so there is no point in pointing out their location.

Reply:
Frankly speaking, I am ignorant about this dummy_cache thing (and did not use it in antares plugin)
It is required in configuration, as you stated earlier. It is also present in the dispather config itself. But why, if generally no dummy products are provided? 
And should this fields then concide  in all configs?

Reply:
We placed them in containers when suitable. In generally, it might be useful to still have it, to enable testing when backend is not available. It might be redesigned though, it's kind of rigid.